designs/Verify-Verbosity.md

```diff
@@ -0,0 +1,240 @@
+`Certificate -> Service index URL (If applicable)`| ❌       | ✔️          | ✔️         | ✔️         | ✔️   
+```
+Once this spec has been implemented the output of `nuget.exe verify` command for various verbosity levels will change and be in sync with `dotnet nuget verify` command.
+```
+
+### Log level mapping - The following details are copied from [here](https://github.com/NuGet/Home/blob/dev/designs/Package-List-Verbosity.md#log-level-mapping). Thanks to [Joel Verhagen](https://github.com/joelverhagen) for the detailed information.
+
+The provided `--verbosity` value will include to NuGet log messages with the following levels:
+
+​             | `q[uiet]` | `m[inimal]` | `n[ormal]` | `d[etails]` | `diag[nostic]`
+------------- | --------- | ----------- | ---------- | ----------- | --------------
+`Error`       | ✔️        | ✔️         | ✔️         | ✔️         | ✔️   
+`Warning`     | ✔️        | ✔️         | ✔️         | ✔️         | ✔️   
+`Minimal`     | ❌        | ✔️         | ✔️         | ✔️         | ✔️   
+`Information` | ❌        | ❌         | ✔️         | ✔️         | ✔️   
+`Verbose`     | ❌        | ❌         | ❌         | ✔️         | ✔️   
+`Debug`       | ❌        | ❌         | ❌         | ✔️         | ✔️   
+
+Note that MSBuild itself has the following mapping for it's own "log levels"
+([source](https://docs.microsoft.com/en-us/visualstudio/msbuild/obtaining-build-logs-with-msbuild?view=vs-2019#verbosity-settings)):
+
+​                                     | `q[uiet]` | `m[inimal]` | `n[ormal]` | `d[etailed]` | `diag[nostic]`
+------------------------------------- | --------- | ----------- | ---------- | ----------- | --------------
+Errors                                | ✔️        | ✔️         | ✔️         | ✔️         | ✔️   
+Warnings                              | ✔️        | ✔️         | ✔️         | ✔️         | ✔️   
+High-importance Messages              | ❌        | ✔️         | ✔️         | ✔️         | ✔️   
+Normal-importance Messages            | ❌        | ❌         | ✔️         | ✔️         | ✔️   
+Low-importance Messages               | ❌        | ❌         | ❌         | ✔️         | ✔️   
+Additional MSBuild-engine information | ❌        | ❌         | ❌         | ❌         | ✔️   
+
+In other words, you can think of the following log concepts per row as equivalent.
+
+NuGet log level    | MSBuild verbosity switch | MSBuild message type
+------------------ | ------------------------ | --------------------
+`Error`, `Warning` | `q[uiet]`                | Errors and Warnings
+`Minimal`          | `m[inimal]`              | High-importance Messages
+`Information`      | `n[ormal]`               | Normal-importance Messages
+`Verbose`, `Debug` | `d[etailed]`             | Low-importance Messages
+​                   | `diag[nostic]`           | Additional MSBuild-engine information
+
+### Example output
+
+This is output of the invocation of `dotnet nuget verify nuget.common.5.9.0-preview.2.nupkg`
```

Comment:
Could you also include the output for different failure scenarios? For example, say a package has been tampered, a certificate does not chain to a trusted root, the signature doesn't have a trusted timestamp, etc...

Reply:
⚠ Given the Debian issue (https://github.com/NuGet/Home/issues/10491) please add this

Reply:
Thanks @loic-sharma. I added example output for couple of failure scenarios in the latest commit.